bin/install

```diff
@@ -241,6 +241,9 @@ download_and_verify_checksums() {
         export GNUPGHOME="${ASDF_DIR:-$HOME/.asdf}/keyrings/nodejs"
       fi
 
+      # Automatically add needed PGP keys
+      source "$(dirname "$0")/import-release-team-keyring"
```

Comment:
nitpick: no need for `source` here. `"$(dirname "$0")/import-release-team-keyring"` is sufficient on its own.